crates/exec-wasmtime/Cargo.toml

```diff
@@ -7,6 +7,9 @@ authors = ["The Enarx Project Developers"]
 repository = "https://github.com/enarx/enarx"
 license = "Apache-2.0"
 
+[features]
+bench = ["dep:tracing-flame"]
```

Comment:
This is not a default feature, and I don't see this feature being enabled for the exec-wasmtime dependency in enarx's Cargo.toml. Where is this enabled? Is it intended to be manually specified when running benchmarks only on this crate?

Reply:
I see `enarx-exec-wasmtime/bench` there, but shouldn't it also have `enarx-shim-kvm/bench` and `enarx-shim-sgx/bench`?